Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Coupled-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/verification/gates.py

```diff
@@ -52,10 +80,13 @@ def __str__(self):
 
     def matrix(self):
         if self.q == 1:
-            cnot = np.array([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]])
-        elif self.q == 2:
             cnot = np.array([[1, 0, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0], [0, 1, 0, 0]])
+        elif self.q == 2:
+            cnot = np.array([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]])
```

Comment:
If Cirq is already imported, why not using the gate definitions in their library? Similarly, why not using their Statevector simulator tools for computing the state?